backend/src/main/java/capstoneexpensetracker/backend/traveler/TravelerService.java

```diff
@@ -44,5 +37,9 @@ public Traveler updateTravelerById(String id, Traveler traveler) {
         }
         throw new NoSuchElementException("No traveler was found with this id");
     }
+
+    public boolean findTravelerExistById(String id) {
```

Comment:
When you check if the traveller exists in the delete-method und throw an exception to the controller, you have more logic inside the service (like we want) and can omit this method here